### DISCUSSION TITLE
Special tax regime

### DISCUSSION THREAD

**Discussion by marionee** [fattureincloud/api#459] [Q&A]
Dear Support Team,

Is there a way through the API to see if the account has a special tax regime?
From the documentation, I can’t find a way to determine if a company is in the flat-rate regime.

Best regards.

**Comment by valmoz** [comment]
Buongiorno, chiedo scusa per il ritardo.
stiamo attualmente pensando se rendere pubblica un'API dedicata a questa necessità.

Grazie per il suggerimento.
Mauro

**Comment by marionee** [comment]
Buongiorno Mauro

grazie per l'aggiornamento, attendo sue.

**Reply by valmoz** [reply]
Buongiorno @marionee 
abbiamo pubblicato la documentazione per la [GetTaxProfile](https://developers.fattureincloud.it/api-reference/#get-/c/-company_id-/settings/tax_profile). Al momento il metodo non è presente sulle SDK, con il prossimo rilascio andremo ad aggiungere la funzionalità anche sulle varie librerie.

Grazie
Mauro

**Comment by marionee** [comment]
Buonasera @valmoz 
grazie per l’aggiornamento! Attendo il prossimo rilascio con la nuova funzionalità.
